initializr-service/src/main/resources/application.yml

```diff
@@ -1011,28 +1017,24 @@ initializr:
       content:
         - name: Cloud Bus
           id: cloud-bus
-          aliases:
-            - cloud-bus-amqp
-          description: A simple control bus defaulting to RabbitMQ
+          description: A simple control bus using Spring Cloud Stream (requires a binder, e.g. Kafka or RabbitMQ)
           groupId: org.springframework.cloud
-          artifactId: spring-cloud-starter-bus-amqp
+          artifactId: spring-cloud-bus
```

Comment:
Same comment on this one. This is a "fake entry"

Reply:
It doesn't have to be, Spring Cloud Bus has a jar file.